candi/bashible/bootstrap/01-base-pkgs.sh.tpl

```diff
@@ -68,55 +96,51 @@ bb-package-fetch-blobs() {
   for PACKAGE_DIGEST in "${!PACKAGES_MAP[@]}"; do
     local PACKAGE_DIR="${BB_FETCHED_PACKAGES_STORE}/${PACKAGES_MAP[$PACKAGE_DIGEST]}"
     mkdir -p "${PACKAGE_DIR}"
-    bb-package-fetch-blob "${PACKAGE_DIGEST}" "${PACKAGE_DIR}/${PACKAGE_DIGEST}.tar.gz"
+
+    retries=0
+    while [ "$retries" -lt 3 ]
```

Comment:
Maybe need using here infinity loop like here https://github.com/deckhouse/deckhouse/blob/a4f2666e26d3f09e937fa404bf3f2f329c3912db/candi/bashible/bashible.sh.tpl#L131

Because without this packages we will not bootstrap node anyway

Reply:
In that case, we could be stuck in an infinite loop, it will be ok?

Reply:
good reason. resolved